README.md

```diff
@@ -1 +1,85 @@
-# Global-Forum-Summit--Capstone-Project
\ No newline at end of file
+# Global-Forum-Summit--Capstone-Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+This project is done using HTML, SCSS and Javascript. The project is also set up with html-css and javascript linters.
+
+## Built With
+
+-HTML
+-CSS
+-Javascript
+-Bootstrap
+
+
+**This is a capstone project done microverse's first module. The content represents an annual event which happens in my city, Sulaimani, through which many international and local issues get discussed by experts.**
+
+## To Get Started
+
+To get a local copy up, CD into the project folder and run:
+
+```
+npm install
+```
+
+## Running Linters From the Command-line
+
+> To execute Webhint linters, run the command bellow (Instead of using . to run the linter from the root, you can also specify a specific directory, i.e.** npx hint ./index**):
+
+```
+npx hint .
+```
+
+Then manually fix the highlighted errors is any.
+
+> To execute style linters, run the command bellow:
+
+```
+npx stylelint "**/*.scss"
+```
+
+Then manually fix the highlighted errors is any.
+
+To auto fix stylelint errors that are fixable automatically, run
+
+```
+npx stylelint "**/*.scss" --fix
+```
+
+You can [ learn more here>>](https://stylelint.io/user-guide/usage/cli/#autofixing-errors)
+
+## Live Demo
+
+[Live Demo Link](https://rozhi-98.github.io/Suli-Forum--Capstone-Project/)
```

Comment:
As for now, your deployment link is not showing the right content, instead, it redirects me to this page:

![image](https://user-images.githubusercontent.com/92395484/198428935-4ed2a8d8-1b94-4597-add4-31bdeb3e2c3a.png)

This can probably be the way you set the deployment in the repository settings. If this is the case, please make sure your deployment branch is this current branch :eye:

![image](https://user-images.githubusercontent.com/92395484/198429707-0f3598ef-dd8b-4857-a931-02ef82cc2ad7.png)

Instead of  `main` in your case should be `Project-branch` :+1: